src/mono/sample/iOS/Program.csproj

```diff
@@ -4,140 +4,57 @@
-      </AotInputAssemblies>
-      <_ExcludeFromAppDir Include="$(_iOSLikeDedupAssembly)" />
-    </ItemGroup>
-
-    <PropertyGroup>
-      <AOTMode Condition="'$(TargetOS)' != 'maccatalyst'">Full</AOTMode>
-      <AOTMode Condition="'$(TargetOS)' == 'maccatalyst' and '$(MonoForceInterpreter)' != 'true'">Full</AOTMode>
-      <AOTMode Condition="'$(TargetOS)' == 'maccatalyst' and '$(MonoForceInterpreter)' == 'true'">JustInterp</AOTMode>
-    </PropertyGroup>
-
-    <MonoAOTCompiler
-        Condition="'$(RunAOTCompilation)' == 'true'"
-        CompilerBinaryPath="@(MonoAotCrossCompiler->WithMetadataValue('RuntimeIdentifier','$(TargetOS)-$(TargetArchitecture.ToLowerInvariant())'))"
-        Mode="$(AOTMode)"
-        OutputType="AsmOnly"
-        Assemblies="@(AotInputAssemblies)"
-        DedupAssembly="$(_iOSLikeDedupAssembly)"
-        AotModulesTablePath="$(AppDir)\modules.m"
-        AotModulesTableLanguage="ObjC"
-        OutputDir="$(PublishDir)"
-        IntermediateOutputPath="$(IntermediateOutputPath)"
-        UseLLVM="$(UseLLVM)"
-        LLVMPath="$(MonoAotCrossDir)">
-        <Output TaskParameter="CompiledAssemblies" ItemName="BundleAssemblies" />
-    </MonoAOTCompiler>
-
-    <ILStrip Condition="'$(RunAOTCompilation)' == 'true' and '$(AOTMode)' == 'Full'" Assemblies="@(BundleAssemblies)" />
-    <PropertyGroup>
-      <RuntimeComponents Condition="'$(RuntimeComponents)' == ''">marshal-ilgen</RuntimeComponents>
+      <AssemblyName>$(AppName)</AssemblyName>
     </PropertyGroup>
+  </Target>
 
-    <AppleAppBuilderTask
-        TargetOS="$(TargetOS)"
-        Arch="$(TargetArchitecture)"
-        ProjectName="$(AppName)"
-        MonoRuntimeHeaders="$(MicrosoftNetCoreAppRuntimePackDir)runtimes\$(TargetOS)-$(TargetArchitecture)\native\include\mono-2.0"
-        Assemblies="@(BundleAssemblies)"
-        MainLibraryFileName="Program.dll"
-        GenerateXcodeProject="True"
-        BuildAppBundle="True"
-        DevTeamProvisioning="$(DevTeamProvisioning)"
-        OutputDirectory="$(AppDir)"
-        Optimized="$(Optimized)"
-        ForceAOT="$(RunAOTCompilation)"
-        ForceInterpreter="$(MonoForceInterpreter)"
-        RuntimeComponents="$(RuntimeComponents)"
-        EnableAppSandbox="$(EnableAppSandbox)"
-        DiagnosticPorts="$(DiagnosticPorts)"
-        StripSymbolTable="$(StripDebugSymbols)"
-        ExcludeFromAppDir="@(_ExcludeFromAppDir)"
-        AppDir="$(MSBuildThisFileDirectory)$(PublishDir)">
-        <Output TaskParameter="AppBundlePath" PropertyName="AppBundlePath" />
-        <Output TaskParameter="XcodeProjectPath" PropertyName="XcodeProjectPath" />
-    </AppleAppBuilderTask>
+  <Import Project="$(MonoProjectRoot)\msbuild\common\LibraryBuilder.props" />
+  <Import Project="$(MonoProjectRoot)\msbuild\apple\build\AppleBuild.props" />
+  <Import Project="$(MonoProjectRoot)\msbuild\apple\build\AppleBuild.InTree.targets" />
```

Comment:
Why do we need `AppleBuild.InTree.targets` and not just `AppleBuild.targets`?

Reply:
That adds a target to override the runtime pack to use the intree one.